CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ Note: This is in reverse chronological order, so newer entries are added to the
 
 Swift v.Next
  -----------
+ * [#3649]
+     Semantic version dependencies can now be resolved against Git tag names that contain only a major and minor component.  A tag with the form `X.X` will be treated as `X.X.0`. This improves compatibility with existing repositories.
```

Comment:
Just 2 nitpicks:

```suggestion
     Semantic version dependencies can now be resolved against Git tag names that contain only major and minor version identifiers.  A tag with the form `X.X` will be treated as `X.X.0`. This improves compatibility with existing repositories.
```

"identifier" is the language used in the semver specification.

```suggestion
     Semantic version dependencies can now be resolved against Git tag names that contain only a major and minor component.  A tag with the form `X.Y` will be treated as `X.Y.0`. This improves compatibility with existing repositories.
```

`X.X` might mislead users to think that the major and minor versions must be equal.

Reply:
These are good suggestions. I’ll make some edits shortly. Sorry if that’s going to reset the CI results again!

Reply:
I made the suggested edits. Thanks for the feedback!